ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3859,3 +3859,33 @@ def test_tensorflow_Zeta(
         x=x[0],
         q=x[1],
     )
+
+
+# Where
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Where",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+        num_arrays=1,
```

Comment:
The default value for this paramters is 1, we can skip it and use the implicit value to make the test shorter.